app/services/service_provider_config.rb

```diff
@@ -0,0 +1,22 @@
+class ServiceProviderConfig
+  def initialize(filename:, issuer:)
```

Comment:
I'm using it here for clarity. There are no hard and fast rules, but we can certainly come up with some. I would say use it when you think there might be confusion about what an argument represents.